_data/retail.yml

```diff
@@ -99,13 +99,13 @@ websites:
       twitter: EBGamesAus
 
     - name: eBay
-      url: http://ebay.com
+      url: https://www.ebay.com
       img: ebay.png
       tfa: Yes
-      hardware: Yes
-      software: Yes
+      sms: Yes
       exceptions:
-          text: "eBay only supports 2FA in some countries."
+          text: "eBay now only supports SMS for 2FA."
+      doc: https://signinalert.ebay.com/sgninalrt/twofa/onboard?flow=myebay
```

Comment:
This doc link requires logging in to eBay to view, which we typically don't allow. I'd recommend we remove it unless eBay can give us a public-facing doc link.